src/main/java/com/example/employee/exceptionHandler/EmployeeExceptionHandler.java

```diff
@@ -0,0 +1,29 @@
+package com.example.employee.exceptionHandler;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.HandlerMapping;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@RestControllerAdvice
+public class EmployeeExceptionHandler extends ResponseEntityExceptionHandler {
+
+    @ExceptionHandler(value = {IllegalArgumentException.class})
+    protected ResponseEntity<Object> handleConflict(RuntimeException ex, WebRequest request) {
+
+        String pathVariables = (String) request.getAttribute(HandlerMapping.PATH_WITHIN_HANDLER_MAPPING_ATTRIBUTE, 0);
+
+        String[] parts = pathVariables.split("/");
+        String firstPath = parts[1];
+        String secondPath = parts[2];
+
+        String bodyOfResponse = String.format("Employee with %s_id %s doesn't exist!", firstPath, secondPath);
```

Comment:
I don't think we need error message here since we already have it when we throw exception

Reply:
I removed the message from .orElseThrow method since the string here is required argument to return object by handleExceptionInternal method.